src/main/drivers/pwm_output.c

```diff
@@ -402,13 +404,15 @@ void pwmWriteBeeper(bool onoffBeep)
 
 void pwmToggleBeeper(void)
 {
+        if(!beeperPwm.io)
```

Comment:
This is not needed, since the check is already done as part of the call to `pwmWriteBeeper`.